mmaction/datasets/pipelines/augmentations.py

```diff
@@ -697,6 +697,141 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class ColorJitter(object):
+    """Randomly change the brightness, contrast, saturation and hue of images.
+
+    If specified to apply color space augmentation, it will distort the image
+    color space by changing brightness, contrast and saturation. Then, it will
+    add some random distort to the images in different color channels.
+    Note that the input images should be in original range [0, 255] and in RGB
+    channel sequence.
+
+    Required keys are "imgs", added or modified keys are "imgs", "eig_val",
+    "eig_vec", "alpha_std" and "color_space_aug".
+
+    Args:
+        color_space_aug (bool): Whether to apply color space augmentations. If
+            specified, it will change the brightness, contrast, saturation and
+            hue of images. Default: False.
+        alpha_std (float): Std in the normal Gaussian distribution of alpha.
+        eig_val (np.ndarray | None): Eigenvalues of [1 x 3] size for RGB
+            channel jitter. If set to None, it will use the default
+            eigenvalues. Default: None.
+        eig_vec (np.ndarray | None): Eigenvectors of [3 x 3] size for RGB
+            channel jitter. If set to None, it will use the default
+            eigenvectors. Default: None.
+    """
+
+    def __init__(self,
+                 color_space_aug=False,
+                 alpha_std=0.1,
+                 eig_val=None,
+                 eig_vec=None):
+        if eig_val is None:
+            # note that the data range should be [0, 255]
+            self.eig_val = np.array([55.46, 4.794, 1.148])
+        else:
+            self.eig_val = eig_val
+
+        if eig_vec is None:
+            self.eig_vec = np.array([[-0.5675, 0.7192, 0.4009],
+                                     [-0.5808, -0.0045, -0.8140],
+                                     [-0.5836, -0.6948, 0.4203]])
+        else:
+            self.eig_vec = eig_vec
+
+        self.alpha_std = alpha_std
+        self.color_space_aug = color_space_aug
+
+    @staticmethod
+    def brightnetss(imgs, delta):
+        """Brightness distortion."""
+        if random.uniform(0, 1) > 0.5:
+            delta = np.array(delta).astype(np.float32)
+            imgs = imgs + delta
+        return imgs
+
+    @staticmethod
+    def contrast(imgs, alpha):
+        """Contrast distortion"""
+        if random.uniform(0, 1) > 0.5:
+            alpha = np.array(alpha).astype(np.float32)
+            imgs = imgs * alpha
+        return imgs
+
+    @staticmethod
+    def saturation(imgs, alpha):
+        """Saturation distortion."""
+        if random.uniform(0, 1) > 0.5:
+            gray = imgs * np.array([0.114, 0.587, 0.299]).astype(np.float32)
+            gray = np.sum(gray, 2, keepdims=True)
+            gray *= (1.0 - alpha)
+            imgs = imgs * alpha
+            imgs = imgs + gray
+        return imgs
+
+    @staticmethod
+    def hue(img, alpha):
+        """Hue distortion"""
```

Comment:
what does alpha mean

Reply:
to control the rotation degree for converting. Add docstring for it